app/src/main/java/com/heapixLearn/discovery/Service/DBPostAdapter.java

```diff
@@ -0,0 +1,31 @@
+package com.heapixLearn.discovery.Service;
+
+import com.heapixLearn.discovery.Entity.ViewablePost;
+
+import java.util.List;
+
+public class DBPostAdapter implements DBPostAdapterI {
```

Comment:
@Scrollend723 what for?

Reply:
https://developer.android.com/studio/write/java8-support